quadlets/cfe18mariadb.container

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Database container for CFE_18
+
+[Container]
+Image=docker.io/mariadb:11.5
+SecurityLabelDisable=true
+# Should be stored as a secret when actually using to deploy.
+Environment=MARIADB_ROOT_PASSWORD=password
+# Static IP for cfe_18 to connect to
+IP=10.88.0.2
```

Comment:
Similar to what @kortemik posted in https://github.com/teragrep/cfe_18/pull/97#discussion_r1930544443

Where is this sourced from?